deploy/runtools/firesim_topology_elements.py

```diff
@@ -325,6 +326,11 @@ def get_required_files_local_paths(self):
         all_paths.append([self.server_hardware_config.get_local_driver_path(), ''])
         all_paths.append([self.server_hardware_config.get_local_runtime_conf_path(), ''])
         all_paths.append([self.server_hardware_config.get_local_assert_def_path(), ''])
+
+        # shared libraries
+        all_paths.append(["$RISCV/lib/libdwarf.so", "libdwarf.so.1"])
```

Comment:
I think it might be nicer to just provide a hook to delivered shared libraries, and then also deliver FESVR along with this, since there's no need to statically link it if we have to deliver stuff like this anyways.

Reply:
made an issue for this